src/GrpcProtos/capture.proto

```diff
@@ -544,6 +544,15 @@ message ThreadStateSlice {
 
   uint64 duration_ns = 6;
   uint64 end_timestamp_ns = 5;
+
+  uint32 wakeup_tid = 7;
+  uint32 wakeup_pid = 8;
+  enum WakeupReason {
+    kNotApplicable = 0;
+    kBlocked = 1;
```

Comment:
Nit: I would call that `kUnblocked`. Looking from the wakeup perspective this makes more sense to me because the reson for waking up is that the thread got unblocked.

Reply:
(Same for the C++ enum)